sdk/state/state.go

```diff
@@ -173,6 +173,20 @@ func (c *Channel) verifySigned(tx *txnbuild.Transaction, sigs []xdr.DecoratedSig
 	return false, nil
 }
 
+func appendNewSignatures(oldSignatures []xdr.DecoratedSignature, newSignatures []xdr.DecoratedSignature) []xdr.DecoratedSignature {
+	m := make(map[string]bool)
+	for _, os := range oldSignatures {
+		m[string(os.Signature)] = true
+	}
+
+	for _, ns := range newSignatures {
+		if _, found := m[string(ns.Signature)]; !found {
```

Comment:
Because you've made the slice value a bool, you can simply do:
```suggestion
		if !m[string(ns.Signature)] {
```

Reply:
https://github.com/stellar/experimental-payment-channels/pull/88/commits/35496032fe6eb27173c1ac3ea4bd2ca6cb2823e5